src/esp/io/JsonEspTypes.cpp

```diff
@@ -103,8 +106,9 @@ bool fromJsonValue(const JsonGenericValue& obj,
     }
   }
 
-  io::readMember(obj, "lightsChanged", keyframe.lightsChanged);
-  if (keyframe.lightsChanged) {
+  itr = obj.FindMember("lights");
```

Comment:
Hmmm, I'm not thrilled about this. I have a dream of one day implementing a new backend for `io::addMember` and `io::readMember` (e.g. serialize to binary, or some other format). So I generally don't like to see us using JSON-specific APIs like `FindMember` inside these functions.

I realize I'm being kinda picky. Let's discuss over VC soon and figure out the best way forward.